arch/sim/src/sim/sim_uart.c

```diff
@@ -676,6 +693,30 @@ static bool tty_txempty(struct uart_dev_s *dev)
 
 void sim_uartinit(void)
 {
+#ifdef CONFIG_SIM_RAM_UART0
+#  ifdef CONFIG_SIM_RAM_UART0_SLAVE
+  sim_uartram_register("/dev/tty0", true);
+#  else
+  sim_uartram_register("/dev/tty0", false);
+#  endif
+#endif
+
+#ifdef CONFIG_SIM_RAM_UART1
+#  ifdef CONFIG_SIM_RAM_UART1_SLAVE
+  sim_uartram_register("/dev/tty1", true);
+#  else
+  sim_uartram_register("/dev/tty1", false);
+#  endif
+#endif
+
+#ifdef CONFIG_SIM_RAM_UART2
+#  ifdef CONFIG_SIM_RAM_UART2_SLAVE
+  sim_uartram_register("/dev/tty2", true);
+#  else
+  sim_uartram_register("/dev/tty2", false);
+#  endif
+#endif
+
```

Comment:
I think it is not a good idea to use tty0, tty1 and tty2 because these device names normally are reserved on Unix/Linux systems to virtual consoles. Maybe something like ttyVS0 (to virtual serial) or ttyVU0 (to virtual UART_

Reply:
Done